docs/source/installation/login.md

```diff
@@ -94,46 +104,47 @@ qhub keycloak -c qhub-config.yaml adduser <username> <password>
 
-Groups represent a collection of users that perform similar actions and therefore require the similar permissions. By default, QHub is deployed with the following groups: 'admin',
-'developer', 'analyst' and 'viewer'.
+Groups represent a collection of users that perform similar actions and therefore require similar permissions. By default, QHub is deployed with the following groups: 'admin',
+'developer', 'analyst' and 'viewer' (in roughly descending order of power).
 
 | Group       | Access to QHub Resources                                                                    |
 | ----------- | ------------------------------------------------------------------------------------------- |
-| 'admin'     | Conda-Store Admin <br> Dask Admin <br> Jupyterhub Admin <br> Graphana Admin                 |
-| 'analyst'   | Conda-Store Developer <br> Jupyterhub Developer <br> Graphana Viewer                        |
-| 'developer' | Conda-Store Developer <br> Dask Developer <br> Jupyterhub Developer <br> Graphana Developer |
+| 'admin'     | Conda-Store Admin <br> Dask Admin <br> Jupyterhub Admin <br> Grafana Admin                 |
+| 'developer' | Conda-Store Developer <br> Dask Developer <br> Jupyterhub Developer <br> Grafana Developer |
+| 'analyst'   | Conda-Store Developer <br> Jupyterhub Developer <br> Grafana Viewer                        |
 
 To create new groups or modify (or delete) existing groups, log in as `root` and click 'Groups' on the left-hand side.
 
-As an example, we create a new group named `conda-store-manager` and this group will have administrator access to the Conda-Store service.
+As an example, we create a new group named `conda-store-manager`. This group will have administrator access to the Conda-Store service.
 
-![Keycloak group tab screenshot - user goups view](../images/keycloak_groups.png)
+![Keycloak groups tab screenshot - user groups view](../images/keycloak_groups.png)
 
 To create a new group, click 'New' in the upper-right hand corner. First, give the new group an appropriate name.
 
 ![Keycloak add group form - name field set to conda-store-manager](../images/keycloak_new_group1.png)
 
-Then under 'Role Mapping', add the appriopriate 'Client Roles' as needed; there should be no need to update the 'Realm Roles'. In this example, the new group only has one mapped
-role however it's possible to attached multiple 'Client Roles' to a single group.
+Then under 'Role Mapping', add the appriopriate 'Client Roles' as needed; there should be no need to update the 'Realm Roles'.
+
+![Keycloak group conda-store-manager form - role mappings tab focused with expanded client roles dropdown](../images/keycloak_new_group2.png)
 
-![Keycloak group conda-store-manager form - role mappings tab focused with expanded client roles  dropdown](../images/keycloak_new_group2.png)
+In this example, the new group only has one mapped role (`conda_store_admin`); however it's possible to attached multiple 'Client Roles' to a single group.
```

Comment:
The relationship between groups and roles here is not at all intuitive if you don't have a background in user permission systems, I think.

Reply:
I agree this is a tough topic as some of those groups as acting as a permission system and some as user "groups" only. c.c @costrouc here to better elucidate this if needed.